### ISSUE TITLE
Queries sensitive to leading whitespace

### ISSUE THREAD

**Issue by randyzwitch** [heavyai/heavydb#222]
Ref: https://github.com/mapd/pymapd/issues/80

When using pymapd, the Calcite backend rejects queries with leading spaces. Trimming query string prior to submission (similar to how mapdql does) would likely fix this issue across the board

**Comment by mflaxman10** [comment]
This one cost me some time...was pasting the same query into mapdql where it worked - confirmed still present, Mar 25th, 2019

**Comment by randyzwitch** [comment]
@mflaxman10 Where did this happen to you, pymapd?

**Comment by mflaxman10** [comment]
Yes, but unfortunately I didn’t record which version and have 0.3 - 0.9  all running within the community server

**Comment by randyzwitch** [comment]
I'll re-open the pymapd issue and see about fixing it there, and then hopefully it gets fixed directly in calcite soonish

**Comment by mike4code** [comment]
Also, fails on comment characters in SQL scripts.  It would really be nice to be able to include comments your SQL scripts and ingest that script into omni for execution.   Common comment characters are -- and /* sql code here */ fail when piped to omnisql.   The following fails.

script-with-comments.sql:
  -- cleanup
  drop table if exists mytmptable;
  -- create the table
  create table mytmptable (id int);

/home/ubuntu/installs/omnisci/bin/omnisql -db test -u mapd -p HyperInteractive < script-with-comments.sql

**Comment by psilvest** [comment]
This issue has been inactive for a while, and the project has changed substantially. If this still occurs on the latest release, please add reproduction steps and environment details.
